### PR TITLE
Add glue for pouchdb-find methods: query, createIndex, deleteIndex, and getIndexes.

### PR DESCRIPTION
**Pull request by nolanlawson**
Rebased version of #47.

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -2046,6 +2047,193 @@ function tests(dbName, dbType) {
           ]
         });
       });
-    }); 
-  
+    });
+
+  describe(dbType + ': indexes and queries', function () {
```
Comment (nolanlawson) [inline test/test.js]: These tests will fail against CouchDB unless we are using the master branch. We should probably switch to using the master branch in Travis, e.g. via Docker.

**Comment by nolanlawson** [comment]
This is failing in Travis due to lack of CouchDB 1.7/2.

**Comment by jure** [comment]
I would love to see this one implemented, as it covers my exact use case. Please let me know how I can help!

**Comment by dehuszar** [comment]
+1.  I could use this functionality right now for a project.  Anything I can do to help?

**Comment by aexmachina** [comment]
Me too, any suggestions on how we can move this forward?

**Comment by nolanlawson** [comment]
Someone would need to write a plugin that lives on top of relational-pouch and pouchdb-find and adds in the functionality. I think it's beyond the scope of this project to add support for pouchdb-find just yet.

**Pull request closed without merging**